### PR TITLE
chore: add plugin, cli components to feature template

### PR DESCRIPTION
**Pull request by staycoolcall911**
*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 1]
```diff
@@ -53,6 +53,8 @@ body:
     - SDK
     - Wing Console
     - IDE Extension
+    - CLI
+    - Plugin
```
Comment (eladb) [inline .github/ISSUE_TEMPLATE/feature.yml:57]: What is "plugin"?

**Review by eladb — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature.yml (hunk 1 above)
Comment (staycoolcall911) [reply .github/ISSUE_TEMPLATE/feature.yml:57]: Wing Plugin System. E.g. for this issue: [Enable Plugins to Explicitly State Compatibility #1474](https://github.com/winglang/wing/issues/1474)

Comment (eladb) [reply .github/ISSUE_TEMPLATE/feature.yml:57]: So maybe "Plugins"?

🤷🏻‍♀️

Comment (staycoolcall911) [reply .github/ISSUE_TEMPLATE/feature.yml]: ok, done

**Comment by Chriscbr** [comment]
Would it make sense to update the PR linter scopes as well / keep similar items between them?

https://github.com/winglang/wing/blob/466751d6ab922df6191563f8d147bdad2073a43e/.github/workflows/pull-request-lint.yml#L22-L31

**Review by eladb — APPROVED** [review]

**Comment by staycoolcall911** [comment]
> Would it make sense to update the PR linter scopes as well / keep similar items between them?
> 
> https://github.com/winglang/wing/blob/466751d6ab922df6191563f8d147bdad2073a43e/.github/workflows/pull-request-lint.yml#L22-L31

Good catch @Chriscbr - I added this scope to the pr linter and docs: https://github.com/winglang/wing/pull/1494

**Pull request merged**